Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -854,9 +870,12 @@ ErrorOr<void> VirtualFileSystem::rmdir(Credentials const& credentials, StringVie
         return {};
```

Comment:
Unrelated, but this can probably be optimized to break out the loop when child_child > 2